### ISSUE TITLE
[Bug]: importing previously exprted svg changes clipPath

### ISSUE THREAD

**Issue by hobyte** [fabricjs/fabric.js#9406]
### CheckList

- [X] I agree to follow this project's Code of Conduct
- [X] I have read and followed the [Contributing Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md)
- [X] I have read and followed the [Issue Tracker Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md#%EF%B8%8F-issue-tracker)
- [X] I have searched and referenced existing issues and discussions
- [X] I am filing a **BUG** report.
- [X] I have managed to reproduce the bug **after** upgrading to the **latest** version
- [X] I have created an accurate and minimal reproduction

### Version

5.3.0

### In What environments are you experiencing the problem?

Firefox

### Node Version (if applicable)

None

### Link To Reproduction

https://gitlab.com/-/snippets/3608017

### Steps To Reproduce

1. open gitlab snippet
2. click on reload under one of the images. This exports the image from the canvas as svg, clears the canvas and reimports the image

### Expected Behavior

Image should look the same

### Actual Behavior

- Images without `clipPath` (2. and 4.) are the same after import. 
- Images with `clipPath` (1. and 3.) change drastically. Parts of the image or the whole image will be invisible.
![grafik](https://github.com/fabricjs/fabric.js/assets/63231858/aba5fe0c-c021-4b78-8fd3-741faeb3a657)

**Comment by asturur** [comment]
Didn't dig super deep but seems a legit bug to me.
In general toSVG / fromSVG is not suggest for import/export but that is especially for text.

I would expect clipPath to work ( unless i forgot of some hard issue that has been abandoned )

**Comment by asturur** [comment]
<img width="1445" alt="image" src="https://github.com/fabricjs/fabric.js/assets/1194048/5014be48-2285-4d9b-9d39-31ed71641d8c">

the exported SVG after loading on canvas is still good.
So we output something we are not able to parse back

**Comment by schtucks** [comment]
I'm experiencing a similar issue with fabric.js version `5.3.0`. I'm repeatedly exporting an SVG image from a canvas and then importing it back using the following methods:

Exporting: `canvas.toSVG(svgData)`
Importing: `fabric.loadSVGFromString(svgString)`

The initial import works correctly, but subsequent export-import cycles cause problems with my drawing area's properties. It appears that the original clipPath on my canvas is somehow being transferred to the drawing area. It seems like this transfer results in the imported image shrinking drastically with each cycle until it completely disappears.

For context, my drawing area is a `fabric.Image` object centered on the canvas. The canvas has a clipPath with dimensions matching the drawing area, which prevents users from drawing outside its boundaries. So there are just two objects on my canvas.
I've included screenshots to illustrate this behavior:

 1st import:
![image](https://github.com/user-attachments/assets/8079f880-c52f-4bcb-9502-1b9834afaa95)

2nd import:
![image](https://github.com/user-attachments/assets/f3aa23c2-da55-4975-966e-8db374a88c99)

3rd import: 
![image](https://github.com/user-attachments/assets/ffda0bd7-39a1-438f-be68-7809f07f1694)

I'm seeking assistance in understanding why this is happening and how to prevent it, maintaining the image's original size and position through multiple export-import cycles.

**Comment by asturur** [comment]
Yes the clipPath on the canvas doesn't exists in svg so it ends up being on the objects.
Apart this use case, there was a legit bug in this issue that hasn't been addressed

**Comment by schtucks** [comment]
In my application, I fixed the issue by identifying the drawing area using an ID when it is read from loadSVGFromString. I manually delete the clipPath that is added during the import process. After that, I add a new clipPath to my canvas that matches the dimensions of the drawing area.

Furthermore, I now identify all objects using custom IDs and only add those that have an ID to the canvas. This approach helps me avoid the issue of unwanted clipPaths being added.

Additionally, when adding the existing objects, I also clean up the clipPath for each object before adding them to the canvas.
